Automattic/woocommerce-payments
# Fix Documents page loading on WooCommerce 8.2.0

Fixes #7461 

#### Changes proposed in this Pull Request

* Includes the `wp-api-request` core script to allow the use of localized version of `wpApiSettings` for when the Documents page is loaded. `wpApiSettings` is used to build the urls to each document in the list.

#### Testing instructions

* Make sure you are running WooCommerce v8.2.0.
* Navigate to Payments > Documents and confirm you receive no error.
  * This menu can be enabled in WCPay Dev Tools if not enabled.

-------------------


**Post merge**
